app/controllers/passthrough_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PassthroughController < ApplicationController
```

Comment:
I'm not convinced of the name `Passthrough` or the need for a separate controller.
I think we could just as easily add an action to documents_controller to achieve this.